BaseClasses.py

```diff
@@ -690,7 +690,7 @@ def all_done() -> bool:
             if not sphere:
                 # ran out of places and did not finish yet, quit
                 logging.warning(f"Could not access required locations for accessibility check."
-                                f" Missing: {locations}")
+                                f" Unreachable Progression Locations: {locations}")
```

Comment:
"Progression" isn't *quite* accurate since locations accessibility checks *all* locations, and minimal will skip plenty of progression locations. I can give or take title casing this but it's inconsistent.
```suggestion
                                f" Unreachable required locations: {locations}")
```

Reply:
I think you didn't get the point of this PR. The locations listed here are the set of locations that contain progression and weren't reached during that test, which is a point of confusion, as, as you said, minimal skips over a lot of progression.

I prefer giving an example, say you don't pass accessibility and the boss key happen to be in the boss room (and nothing else relevant is beyond that door), then minimal will list here that the boss reward containing that boss key is NOT reachable. I'm vehemently AGAINST saying that this location is required, because it isn't

Reply:
>The locations listed here are the set of locations that contain progression
This is false and exactly my point.
![Screenshot 2025-03-28 194015](https://github.com/user-attachments/assets/e88a7a6b-dbc2-4848-8f95-0d4ae86f7096)

Reply:
Oh, it's only the case in minimal, huh

Reply:
Unreachable Potentially Required Locations ?

Reply:
I'll concede that required is inaccurate, but progression is more inaccurate since full accessibility is the default.